contracts/PetPark.sol

```diff
@@ -1,7 +1,90 @@
-//SPDX-License-Identifier: Unlicense
+// SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
 
 contract PetPark {
 
-}
\ No newline at end of file
+    address owner; 
+
+    enum AnimalType {
+        NONE,
+        Fish,
+        Cat,
+        Dog,
+        Rabbit,
+        Parrot
+    }
+
+    enum Gender {
+        Male,
+        Female
+    }
+
+    struct Borrower {
+        AnimalType animal;
+        bool isBorrowing;
+        Gender gender;
+        uint age;
+    }
+    
+    modifier onlyOwner(){
+        require(msg.sender == owner, "Not an owner");
+        _;
+    }
+   
+    event Added(AnimalType _animalType, uint count);
+    event Borrowed(AnimalType _animalType);
+    event Returned(AnimalType _animalType);
+
+    mapping(AnimalType => uint) animalCount;
+    mapping(address => Borrower) public borrowers;
+
+    constructor(){
+        owner = msg.sender;
+
+    }
+
+    function add(AnimalType _animalType, uint count) public onlyOwner {
+        require(uint(_animalType) > 0, "Invalid animal");
+        animalCount[_animalType] = animalCount[_animalType] + count;
+        emit Added(_animalType, count);
+    }
+
+    function animalCounts(AnimalType _animalType) public view returns (uint) {
```

Comment:
Since solidity public mappings are automatically exposed as functions, you can just make `animalCounts` as a public mapping

Reply:
I figured that out the other day.. thanks for catching this